data/tx/legislature/Mayes-Middleton-1399b6ba-cbfa-45ca-bf91-6ba89fe777bf.yml

```diff
@@ -1,24 +0,0 @@
-id: ocd-person/1399b6ba-cbfa-45ca-bf91-6ba89fe777bf
-name: Mayes Middleton
-given_name: Mayes
-family_name: Middleton
-gender: Male
-image: https://house.texas.gov/photos/members/3645.jpg
-party:
-- name: Republican
-roles:
-- type: lower
-  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
-  district: '23'
-offices:
-- classification: capitol
```

Comment:
@sroomf this should be undeleted and the roles should be:
- end_date: '2023-01-10'
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '23'
- type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '11'